src/libstd/sys/unix/ext/fs.rs

```diff
@@ -32,8 +32,27 @@ pub trait FileExt {
     ///
     /// The current file cursor is not affected by this function.
     ///
-    /// Note that similar to `File::read`, it is not an error to return with a
+    /// Note that similar to [`File::read`], it is not an error to return with a
     /// short read.
+    ///
+    /// [`File::read`]: ../../../../std/fs/struct.File.html#read.v
```

Comment:
Why did you use `read.v` instead of `method.read`? I know they go to the same spot but i'm more used to reading the `method.` style link anchors. (Same for `write` farther down.)

Reply:
It's not targetting the same thing actually. :)

Reply:
I would consider "the first child element of a container versus the container itself" to be close enough.

![image](https://user-images.githubusercontent.com/5217170/32497572-9a89a52e-c392-11e7-8f7b-832dd1ceade5.png)

Reply:
Hum, good point.